samples/Samples.AzureFunctions/GraphQL.cs

```diff
@@ -0,0 +1,34 @@
+#pragma warning disable IDE0060 // Remove unused parameter
+
+using GraphQL.Server.Transports.AspNetCore.AzureFunctions;
+using GraphQL.Server.Ui.Playground;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Samples.AzureFunctions;
+
+public class GraphQL
+{
+    [FunctionName("GraphQL")]
+    public static IActionResult RunGraphQL(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest request,
+        ILogger log)
+    {
+        log.LogInformation("C# HTTP trigger function processed a GraphQL request.");
+
+        return new AzureGraphQLActionResult(request);
+    }
+
+    [FunctionName("Playground")]
+    public static IActionResult RunPlayground(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequest request,
+        ILogger log)
+    {
+        log.LogInformation("C# HTTP trigger function processed a request for the GraphQL Playground UI.");
+
+        return new PlaygroundActionResult(opts => opts.GraphQLEndPoint = "/api/graphql");
```

Comment:
> /api/graphql

Where is it configured?

Reply:
I don't know much about Azure Functions.  I created the sample and renamed the `[FunctionName]` and that's the url it was at.  It appears in the console app when you start debugging.

Reply:
I mean I did not find the code that starts GraphQL endpoint at `/api/graphql`.

Reply:
Please see line 53 and consult with Azure Functions documentation here:

https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=in-process%2Cfunctionsv2&pivots=programming-language-csharp#customize-the-http-endpoint

Reply:
Ah! )